pkg/awsutils/awsutils.go

```diff
@@ -633,6 +642,7 @@ func (cache *EC2InstanceMetadataCache) getENIMetadata(eniMAC string) (ENIMetadat
 	}
 	ipv4Available := false
 	ipv6Available := false
+	networkCard := 0
```

Comment:
Is there a way to avoid tackling the network card info? I'm thinking we want to skip the whole network cards / device > 1 thingy. Just focus on disabling api calls.